rust/util_libs/db/src/schemas.rs

```diff
@@ -35,9 +35,13 @@ pub enum UserPermission {
 
 #[derive(Serialize, Deserialize, Clone, Debug, Default)]
```

Comment:
```suggestion
#[derive(Serialize, Deserialize, Clone, Debug, Default)]
#[serde(default)] 
```

i believe putting this on the container will be enough to replace all the field occurrences of this, except for the ones that don't have an `impl Default`.